core/templates/dev/head/pages/topics_and_skills_dashboard/create_new_skill_modal_directive.html

```diff
@@ -0,0 +1,15 @@
+<div class="modal-header">
+  <h3>Create New Skill</h3>
+</div>
+
+<div class="modal-body">
+  <div class="oppia-rule-details-header">
+    <strong>New Skill Description</strong>
+    <input type="text" class="form-control" ng-model="newSkillDescription" />
```

Comment:
Hi @tjiang11 and @seanlip , shouldn't this `input` tag be not closed, that is,  `<input type="text" class="form-control" ng-model="newSkillDescription">`?
The html indent function, which also validates the html raises an exception here:
```bash
__main__.TagMismatchException: Error in line 8 of file core/templates/dev/head/pages/topics_and_skills_dashboard/create_new_skill_modal_directive.html
```
(From Travis logs)

Thanks!

Reply:
Thanks @apb7! @tjiang11 -- worth figuring out why this didn't trigger on your machine?

@apb7 these html checks do get run on presubmit right?